src/_pytest/debugging.py

```diff
@@ -74,7 +74,7 @@ class pytestPDB:
 
     _pluginmanager = None
     _config = None
-    _saved = []
+    _saved: list = []
```

Comment:
this isn't a useful annotation -- when mypy says "need annotation for ..." it's requesting that you fill out the generic parameter (it already knows it is a list, but "what list?")

Reply:
Agreed, but I wasn't sure about this one -- currently I just want to get it running, so did the quick fix. When we start rejecting `Any` we'll need to confront this.